docs/cn/core-services/Caching.md

```diff
@@ -1,259 +1,507 @@
+来为经常访问的数据征回空间([块注释策略](#block-annotation-policies))。
+在同一文件中不同的块可能根据访问频率不同而具有不同数量副本。
 
-被动TTL使用以下配置选项:
+默认情况下，此复制或征回决定以及相应的数据传输
+对访问存储在Alluxio中数据的用户和应用程序完全透明。
 
-* `alluxio.user.file.load.ttl` - 针对从底层存储加载到Alluxio中的任何新文件的默认生存时间。默认情况下没有TTL。
-* `alluxio.user.file.load.ttl.action` - 任何ttl设置的默认操作，设置在从一个under store加载到Alluxio的文件上。
-默认情况下是 `DELETE`。
-* `alluxio.user.file.create.ttl` - 新创建的任何文件的默认的生存时间。默认情况下没有TTL。
-* `alluxio.user.file.create.ttl.action` - 在一个新创建的文件上的任何ttl设置的默认操作。默认情况下是`DELETE`。
+### 主动复制
 
-有两对选项，一组用于  `load`，一组用于`create`。`load`指的是Alluxio从底层存储获取的文件。
-`Create`指创建的新文件或目录。
+除了动态复制调整之外，Alluxio还提供API和命令行
+界面供用户明确设置文件的复制级别目标范围。
+尤其是，用户可以在Alluxio中为文件配置以下两个属性:
 
-这两个选项在默认情况下都是禁用的，应该只由具有严格数据访问模式的客户端启用
-例如，要删除`runTests`在1分钟后创建的文件:
+1.`alluxio.user.file.replication.min`是此文件的最小副本数。
+默认值为0，即在默认情况下，Alluxio可能会在文件变冷后从Alluxio管理空间完全删除该文件。
+通过将此属性设置为正整数，Alluxio
+将定期检查此文件中所有块的复制级别。当某些块
+的复制数不足时，Alluxio不会删除这些块中的任何一个，而是主动创建更多
+副本以恢复其复制级别。
+1.`alluxio.user.file.replication.max`是最大副本数。一旦文件该属性
+设置为正整数，Alluxio将检查复制级别并删除多余的
+副本。将此属性设置为-1为不设上限(默认情况)，设置为0以防止
+在Alluxio中存储此文件的任何数据。注意，`alluxio.user.file.replication.max`的值
+必须不少于`alluxio.user.file.replication.min`。
 
+例如，用户可以最初使用至少两个副本将本地文件`/path/to/file`复制到Alluxio:
+
+```console
+$ ./bin/alluxio fs -Dalluxio.user.file.replication.min=2 \
+copyFromLocal /path/to/file /file
 ```
-bin/alluxio runTests -Dalluxio.user.file.create.ttl=1m -Dalluxio.user.file.create.ttl.action=DELETE
+
+接下来，设置`/file`的复制级别区间为3到5。需要注意的是，在后台进程中完成新的复制级别范围设定后此命令将马上返回，实现复制目标是异步完成的。
+
+```console
+$ ./bin/alluxio fs setReplication --min 3 --max 5 /file
+```
+
+设置`alluxio.user.file.replication.max`为无上限。
+
+```console
+$ ./bin/alluxio fs setReplication --max -1 /file
+```
+
+重复递归复制目录`/dir`下所有文件复制级别(包括其子目录）使用`-R`:
+
+```console
+$ ./bin/alluxio fs setReplication --min 3 --max -5 -R /dir
+```
+
+要检查的文件的目标复制水平，运行
+
+```console
+$ ./bin/alluxio fs stat /foo
+```
+
+并在输出中查找`replicationMin`和`replicationMax`字段。
+
+##检查Alluxio缓存容量和使用情况
+
+Alluxio shell命令`fsadmin report`提供可用空间的简短摘要
+以及其他有用的信息。输出示例如下:
+
+```console
+$ ./bin/alluxio fsadmin report
+Alluxio cluster summary:
+    Master Address: localhost/127.0.0.1:19998
+    Web Port: 19999
+    Rpc Port: 19998
+    Started: 09-28-2018 12:52:09:486
+    Uptime: 0 day(s), 0 hour(s), 0 minute(s), and 26 second(s)
+    Version: 2.0.0
+    Safe Mode: true
+    Zookeeper Enabled: false
+    Live Workers: 1
+    Lost Workers: 0
+    Total Capacity: 10.67GB
+        Tier: MEM  Size: 10.67GB
+    Used Capacity: 0B
+        Tier: MEM  Size: 0B
+    Free Capacity: 10.67GB
+```
+
+Alluxio shell还允许用户检查Alluxio缓存中多少空间可用和在用。
+
+获得Alluxio缓存总使用字节数运行:
+
+```console
+$ ./bin/alluxio fs getUsedBytes
+```
+
+获得Alluxio缓存以字节为单位的总容量
+
+```console
+$ ./bin/alluxio fs getCapacityBytes
 ```
 
-注意，如果您尝试使用这个示例，请确保将 `alluxio.master.ttl.checker.interval` 设置为较短时间，即1分钟。
+Alluxio master web界面为用户提供了集群的可视化总览包括已用多少存储空间。可以在`http:/{MASTER_IP}:${alluxio.master.web.port}/中找到。
```

Comment:
Same as before backticks need to come as pairs. Add the backtick after http:/{MASTER_IP}:${alluxio.master.web.port}/